torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -1153,6 +1153,7 @@ def call_function(
         )
 
         example_value = lowered_module.original_module.module()(*real_sub_args)
+        example_value = deepcopy_to_fake_tensor(example_value, tx.fake_mode)
```

Comment:
em...can you try`with tx.fake_mode:` to wrap around the original_module.module() call?